native/dll/Win32DLL/SageTVWin32DLL.def

```diff
@@ -3,4 +3,3 @@
 LIBRARY      "SageTVWin32"
 
 SECTIONS
```

Comment:
I don't know what changing this does...so unless you can explain to me why you're changing it, please revert this part of the change.

Reply:
When it's there, we get a compile warning of 
`LINK : warning LNK4039: section '.shared' specified with /SECTION option does not exist`
which from my reading is because it's defined in the .def file but not used

Reply:
OK